front/temporal/scrub_workspace/workflows_queue_v1.ts

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
this is a copy/paste of `front/temporal/scrub_workspace/workflows.ts` *but* for ....